app/controllers/cinema_halls_controller.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class CinemaHallsController < ApplicationController
+  before_action :set_cinema_hall, only: %i[show update destroy]
+
+  # GET /cinema_halls
+  def index
+    @cinema_halls = CinemaHall.all
+
+    render json: @cinema_halls
+  end
+
+  # GET /cinema_halls/1
+  def show
+    render json: @cinema_hall
+  end
+
+  # POST /cinema_halls
+  def create
+    @cinema_hall = CinemaHall.new(cinema_hall_params)
+
+    if @cinema_hall.save
+      render json: @cinema_hall, status: :created, location: @cinema_hall
```

Comment:
`location` attribute is not needed for us I guess

Reply:
I'm not yet using `json:api`, but probably will in the future and since it's documentation mentions `location` attribute only briefly:
> The response **SHOULD** include a `Location` header identifying the location of the newly created resource.

not clearly stating what it's exact purpose is, I decided not to meddle with it for now (it was automatically generated). Could you tell me something more why it's superfluous right now?